examples/get-maintenance-json

```diff
@@ -0,0 +1,57 @@
+#!/usr/bin/env python
+import os
+import logging
+from argparse import ArgumentParser
+
+from pubtools.pulplib import Client
+
+log = logging.getLogger("get-maintenance")
+
+
+def get_maintenance(client):
+
+    maintenance = client.get_maintenance_status().result()
+    log.info("maintenance status: %s", maintenance)
```

Comment:
I think this example is probably a bit *too* trivial.  Can it be something more realistic?

Like instead of a raw dump, tell us:

- which repo(s) are in maintenance mode
- who set them in maintenance mode / the comment